Tools/jit/README.md

```diff
@@ -41,6 +41,12 @@ Homebrew won't add any of the tools to your `$PATH`. That's okay; the build scri
 
 Install LLVM 18 [by searching for it on LLVM's GitHub releases page](https://github.com/llvm/llvm-project/releases?q=18), clicking on "Assets", downloading the appropriate Windows installer for your platform (likely the file ending with `-win64.exe`), and running it. **When installing, be sure to select the option labeled "Add LLVM to the system PATH".**
 
+Alternatively, you can use [chocolatey](https://chocolatey.org):
+
+```sh
+choco install llvm --version=18.1.0
```

Comment:
Is it possible to specify just `--version=18`, without minor and patch releases? Other tools do not specify exact version.

Reply:
> Is it possible to specify just `--version=18`, without minor and patch releases? Other tools do not specify exact version.

Unfortunately, no.
We can also omit ``--version`` and the latest stable release will be installed. What do you think about that?

Reply:
So, here's the list of available versions:
```
llvm 18.1.6 [Approved] Downloads cached for licensed users
llvm 18.1.5 [Approved] Downloads cached for licensed users
llvm 18.1.4 [Approved] Downloads cached for licensed users
llvm 18.1.3 [Approved]
llvm 18.1.2 [Approved]
llvm 18.1.0 [Approved] Downloads cached for licensed users
```
Which one do you prefer? The latest one?

Reply:
Latest, yes.

Reply:
Done